### ISSUE TITLE
compile errors with swift 4.1 and xcode 9.3

### ISSUE THREAD

**Issue by benchd** [httpswift/swifter#302]
I am trying to update my app from swift 3 to swift for 4 and getting compile errors on Scopes.swift
on scope() "Missing argument for parameter #1 in call,
public typealias Closure = (Void) -> Void "When calling this function in Swift 4 or later, you must pass a '()' tuple; did you mean for the input type to be '()'?"

I created a small test app with just the swifter pod and getting the same thing.

I am using xcode 9.3, in build settings I have tried setting the Swift Language Version to unspecified and Swift 4.1 but I am getting the same errors when I do a Product clean and then Product Build
My podfile has:
pod 'Swifter', '~> 1.3.3'

**Comment by neosinha** [comment]
I am experiencing the same problem.

**Comment by CristianDRM** [comment]
I noticed that the library was updated to swift in 1.4.0 release, but not published in cocoapods.

**Comment by kimaldis** [comment]
+1. Same compile errors.

**Comment by neosinha** [comment]
I got around by pulling in the source files directly and creating a "group" called "SwifterFiles". Of course we cannot use "import Swifter" anymore but the "HttpServer" object is available directly.

**Comment by muskacirca** [comment]
same error

**Comment by jotaemepereira** [comment]
I've just fixed using the 1.4.2 release. I've added this to my Podfile and it's working:

```
pod 'Swifter', :git => 'https://github.com/httpswift/swifter.git', :tag => '1.4.2']
```

**Comment by Vkt0r** [comment]
A new version was released `1.4.5` please feel free to open it again in the issue persist 😅.
